package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screenshot-promise",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Versions are bumped separately and automatically at release. Don't touch this ;).